webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4016,10 +4017,19 @@ var userOverride = {
                                 callback
                             );
                         } else {
-                            if (MCK_TRIGGER_MSG_NOTIFICATION_TIMEOUT > 0) {
-                                ALStorage.clearMckMessageArray();
+                            if (kommunicate._globals.zendeskChatSdkKey) {
+                                var groupId = result.response[0].id;
+                                $applozic.fn.applozic(
+                                    'loadGroupTab',
+                                    groupId,
+                                    callback
+                                );
+                            } else {
+                                if (MCK_TRIGGER_MSG_NOTIFICATION_TIMEOUT > 0) {
```

Comment:
If msg notification trigger timeout is zero then what should happen?

Reply:
Check this PR: https://github.com/Kommunicate-io/Kommunicate-Web-SDK/pull/980/files
This one fixed here
We will merge this branches.